docs/for-developers.md

```diff
@@ -48,14 +47,14 @@ development server:
 
 #### Import Data
 
-Once the database, rabbitMQ service, importer and the application 
-are running, you're ready to import data. 
-First, [create a Django admin user](https://docs.djangoproject.com/en/2.1/intro/tutorial02/#creating-an-admin-user) 
+Once the database, rabbitMQ service, importer and the application
+are running, you're ready to import data.
+First, [create a Django admin user](https://docs.djangoproject.com/en/2.1/intro/tutorial02/#creating-an-admin-user)
 and log in as that user.
-Then, Click on Campaigns and then Add Campaign. Enter a name for the campaign 
-and project and a loc.gov item URL such as 
-[https://www.loc.gov/item/mss859430011](https://www.loc.gov/item/mss859430011) and click Create.
-
+Then, go to the Admin area (under Account) and click "Bulk Import Items".
+Upload a spreadsheet populated according to the instructions. Once all the import
```

Comment:
Since we don't have a source for that spreadsheet, should we have a simple test one? That might be handy for anyone who doesn't want to download tens of thousands of images, too.

Reply:
Yes, I had the same thought. A small test spreadsheet is a good idea. Populating any test content without a spreadsheet is pretty tedious.